### DISCUSSION TITLE
技术预览版 - 4

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#243] [通知]
## 🐛 Bug 修复

- 修复直播中断的问题 @Richasy (#241)
- 修复PGC内容不能通过年份索引的问题 @Richasy (#238)
- 完善直播消息滚动机制 @Richasy (#223)
- 修复首页推荐刷新错误 @Richasy (#222)
- 修复弹幕加粗设置没有保存正确的问题 @Richasy (#215)
- 修复播放器全屏时的UI问题 @Richasy (#214)

## 🧰 管理与维护

- 修复播放器全屏时的UI问题 @Richasy (#214)
- 创建Release自动化脚本 @Richasy (#207)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2109.2002.pre-release'>技术预览版 - 4</a>.</em>

**Comment by Orajz** [comment]
刚刚关注就有更新提示，就很棒

**Comment by tm0210** [comment]
直播现在可以很快地加载出来，大爱！！！
另外，安装包的大小有点问题，从178M突然变成483M。x64安装包解压后发现x64文件夹里的第一个文件夹里面的东西是安装文件，大小190M。其他四个文件夹好像是其他软件的安装包。
![Uploading 屏幕截图 2021-09-22 182246.png…]()

**Reply by Richasy** [reply]
才注意到……看来管道的配置有点问题

**Comment by ghost** [comment]
作者太勤奋了 大爱

**Comment by 1336841591** [comment]
作者大大，好像看番剧有时候并不记录进度，并且同屏弹幕密度无法更改，还有就是有没有安排上下集的按键呢？

**Reply by Richasy** [reply]
莫催……Issue里有的会慢慢修，有优先级

**Comment by qq956681303** [comment]
额...surfacex更新以后打开没反应了
